lib/jvm.sh

```diff
@@ -134,6 +134,7 @@ install_profile() {
   mkdir -p "$profileDir"
   cp "${bpDir}/opt/jvmcommon.sh" "${profileDir}"
   cp "${bpDir}/opt/jdbc.sh" "${profileDir}"
+  cp "${bpDir}/opt/redis.sh" "${profileDir}"
```

Comment:
`redis.sh` might be too generic of a name, and could collide with a script installed by another buildpack (because they all share `.profile.d` in the slug. Maybe `jvm-redis.sh`?

Reply:
Good point, I'll prefix it.